packages/blade/src/components/Dropdown/DropdownButton.tsx

```diff
@@ -54,17 +53,16 @@ const _DropdownButton = ({
       size={size}
       type={type}
       variant={variant}
-      accessibilityLabel={accessibilityLabel}
       testID={testID}
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       ref={triggererRef as any}
-      {...makeAccessible({
+      accessibilityProps={{
+        label: accessibilityLabel,
         hasPopup: getActionListContainerRole(hasFooterAction, 'DropdownButton'),
         expanded: isOpen,
         controls: `${dropdownBaseId}-actionlist`,
-        role: 'combobox',
```

Comment:
where did the role go?

Reply:
Removed it. It was previously getting overriden to role button and the screenreader seemed to announce it correctly with that too so kept the behaviour same. Will reevaluate what should be the role of trigger during autocomplete